src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -67,7 +67,14 @@ and environment variables.
 The `OtlpExporterOptions` type setters take precedence over the environment variables.
 
 This can be achieved by providing an `Action<OtlpExporterOptions>` delegate to the
-`AddOtlpExporter()` method.
+`AddOtlpExporter()` method or using `AddOptions<OtlpExporterOptions>()`.
+
+If additional services from the dependency injection are required, they can be configured like this:
+```csharp
+services.AddOptions<OtlpExporterOptions>().Configure<Service>((opts, svc) => {
+    // ...
+});
+```
```

Comment:
@cromefire Hey would you be open to moving this into its own PR? We can get this in more quickly than the new public API changes.

Reply:
Sure, I'll move it in the coming days.

Reply:
Converted this PR